doc/tutorial/c_tutorial/reachibility_monitor_p1.c

```diff
@@ -0,0 +1,119 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+#include <string.h> // required for strncmp()
+#include <errno.h>  // required for error code identifiers
+
+#include "ddlog.h"
```

Comment:
This is generated by the compiler in the _ddlog directory.

Reply:
I would point at it explicitly, it has lots of comments and it really defines the API between DDlog and C.